src/components/Button.js

```diff
@@ -58,7 +56,8 @@ class Button extends React.Component {
   };
 
   render () {
-    const styles = this.styles();
+    const theme = StyleUtils.mergeTheme(this.props.theme, this.props.primaryColor);
```

Comment:
yeah i realized after reviewiing a few more of these that you were way ahead of me 🙃